src/libs/PaymentUtils.js

```diff
@@ -71,6 +92,42 @@ function formatPaymentMethods(bankAccountList, cardList, payPalMeData = null) {
     return combinedPaymentMethods;
 }
 
+/**
+ * Gets the selected plaid bank account object from plaidData
+ * @param {Object} personalBankAccount
+ * @param {Object} plaidData
+ * @returns {Object}
+ */
+function getPendingBankAccount(personalBankAccount = {}, plaidData = {}) {
```

Comment:
Why have both the params as optional? Maybe we shouldn't call this function at all if there isn't a personal bank account and plaid data. Then we can remove the multiple `if (_.isEmpty(...))` below

Reply:
I can clean it up